packages/otp2-tile-overlay/src/util.ts

```diff
@@ -8,37 +8,39 @@ const generateFloatingVehicleColor = (formFactor: string) => [
 ];
 
 // eslint-disable-next-line import/prefer-default-export
-export const LAYER_PAINT = {
-  rentalStations: {
-    "circle-color": generateFloatingVehicleColor("formFactors"),
-    "circle-opacity": 0.9,
-    "circle-stroke-color": "#333",
-    "circle-stroke-width": 3
-  },
-  rentalVehicles: {
-    "circle-color": generateFloatingVehicleColor("formFactor"),
-    "circle-opacity": 0.9,
-    "circle-stroke-color": "#333",
-    "circle-stroke-width": 2
-  },
-  stations: {
-    "circle-color": "#fff",
-    "circle-opacity": 0.9,
-    "circle-radius": 10,
-    "circle-stroke-color": "#333",
-    "circle-stroke-width": 3
-  },
-  stops: {
-    "circle-color": "#fff",
-    "circle-opacity": 0.9,
-    "circle-stroke-color": "#333",
-    "circle-stroke-width": 2
-  },
-  vehicleParking: {
-    "circle-color": "black",
-    "circle-opacity": 0.9,
-    "circle-radius": 10,
-    "circle-stroke-color": "#333",
-    "circle-stroke-width": 3
-  }
+export const LAYER_PAINT = (color?: string): any => {
```

Comment:
This should be renamed since it's no longer a constant 
```suggestion
export const generateLayerPaint = (color?: string): any => {
```